src/v2/ion/i18nTransformer.js

```diff
@@ -85,6 +86,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'challenge-authenticator.phone_number.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
   'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
   'challenge-authenticator.okta_verify.credentials.totp': 'oie.okta_verify.totp.enterCodeText',
+  'challenge-authenticator.google_authenticator.credentials.totp': 'oie.google_authenticator.totp.enterCodeText',
```

Comment:
nit: it is otp not totp

Also please add a test like this https://github.com/okta/okta-signin-widget/pull/1602